src/algebra/lattice_ordered_group.lean

```diff
@@ -127,89 +115,62 @@ namespace lattice_ordered_comm_group
 
 -- Bourbaki A.VI.12 Definition 4
 /--
+Let `α` be a lattice ordered commutative group with identity `1`. For an element `a` of type `α`,
+the element `a ⊔ 1` is said to be the *positive component* of `a`, denoted `a⁺`.
+-/
+@[to_additive pos /-"
 Let `α` be a lattice ordered commutative group with identity `0`. For an element `a` of type `α`,
 the element `a ⊔ 0` is said to be the *positive component* of `a`, denoted `a⁺`.
--/
-@[to_additive pos
-  "Let `α` be a lattice ordered commutative group with identity `0`. For an element `a` of type `α`,
-  the element `a ⊔ 0` is said to be the *positive component* of `a`, denoted `a⁺`."]
-def mpos (a : α) : α :=  a ⊔ 1
-postfix `⁺`:1000 := mpos
+"-/,
+priority 100
+] -- see Note [lower instance priority]
+instance has_one_lattice_has_pos_part : has_pos_part (α)  := ⟨λa, a ⊔ 1⟩
 
 /--
+Let `α` be a lattice ordered commutative group with identity `1`. For an element `a` of type `α`,
+the element `(-a) ⊔ 1` is said to be the *negative component* of `a`, denoted `a⁻`.
+-/
+@[to_additive neg /-"
 Let `α` be a lattice ordered commutative group with identity `0`. For an element `a` of type `α`,
 the element `(-a) ⊔ 0` is said to be the *negative component* of `a`, denoted `a⁻`.
--/
-@[to_additive neg
-  "Let `α` be a lattice ordered commutative group with identity `0`. For an element `a` of type `α`,
-  the element `(-a) ⊔ 0` is said to be the *negative component* of `a`, denoted `a⁻`."]
-def mneg (a : α) : α := a⁻¹ ⊔ 1
-postfix `⁻`:1000 := mneg
+"-/,
+priority 100
+] -- see Note [lower instance priority]
+instance has_one_lattice_has_neg_part : has_neg_part (α)  := ⟨λa, a⁻¹ ⊔ 1⟩
```

Comment:
Idem

Reply:
Following the convention you suggested: https://github.com/leanprover-community/mathlib/pull/8673#discussion_r693308231

Reply:
See above.

Reply:
Sorry - hopefully fixed now.